tests/tests/operation_tests.cpp

```diff
@@ -7705,16 +7700,16 @@ BOOST_FIXTURE_TEST_SUITE(auction_window_tests, votes_extended_fixture)
             double allowed_percent_delta = 0.01; // = 1e-2
             double allowed_tokens_delta = 50;
 
-            BOOST_CHECK_EQUAL(fabs(0.25 - voters_reward_percent) < allowed_percent_delta, true);
-            BOOST_CHECK_EQUAL(abs(auw_tokens - total_comment_fund.reward_fund().amount.value) < allowed_tokens_delta, true);
-            BOOST_CHECK_EQUAL(fabs(0.25 - modeled_voters_reward_percent) < allowed_percent_delta, true);
+            APPROX_CHECK_DOUBLE_EQUAL(0.25, voters_reward_percent, allowed_percent_delta);
+            APPROX_CHECK_EQUAL(auw_tokens, total_comment_fund.reward_fund().amount.value, allowed_tokens_delta);
```

Comment:
1. APPROX_CHECK_EQUAL uses <=. Your APPROX_CHECK_DOUBLE_EQUAL uses <. Inconsistency.

2. APPROX_CHECK_DOUBLE_EQUAL is not a better name.